scripts/ci/libraries/_repeats.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Repeat the command up to n times in case of failure
+# Parameters:
+#   $1 - how many times to repeat
+#   $2 - command to repeat (run through bash -c)
+function repeats::repeat_up_to_n_times() {
```

Comment:
I think about the name of this method, and I don't think the word "repeat" describes its use precisely. What do you think of "run_with_retry"?

Reply:
+1 for `run_with_retry`

Reply:
No problem :)